config/envoyconfig/protocols.go

```diff
@@ -41,34 +44,56 @@ var http1ProtocolOptions = &envoy_config_core_v3.Http1ProtocolOptions{
 	},
 }
 
+// Keepalive is a type to enable or disable keepalive
+type Keepalive bool
```

Comment:
Why a typed boolean and not just a normal boolean?

Reply:
I always struggle to remember what random booleans mean in various functions. Isn't Keepalive(true) more readable ?

Reply:
Sorry I don't understand this. I feel like we've had this conversation before. I'll probably keep leaving this comment because it will never make sense to me. Sorry.

I guess we do this all over the place so 🤷 . Sorry for not offering constructive feedback. I cannot defend my perspective.